### PR TITLE
Update views.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
Added some reference material about the PHP syntax for use in template files.  Please let me know if I should change anything!

### CODE REVIEW

File: en/views.rst [hunk 1]
```diff
@@ -66,15 +66,59 @@ View Templates
...
 reply to a remote application via JSON, or output a CSV file for a user.
 
-By default CakePHP template files are written in plain PHP and have a default
-extension of **.ctp** (CakePHP Template). These files contain all the
-presentational logic needed to get the data it received from the
-controller in a format that is ready for the audience you're
-serving to. If you'd prefer using a templating language like
-`Twig <http://twig.sensiolabs.org>`__, a subclass of View will bridge your templating
+CakePHP template files have a default extension of **.ctp** (CakePHP Template)
+and utilize `alternative PHP syntax
```
Comment (ravage84) [inline en/views.rst]: "the" alternative...

File: en/views.rst [hunk 2]
```diff
@@ -66,15 +66,59 @@ View Templates
...
+written in a simplified format as well. The following is an example using
+``foreach``::
+
+  <ul>
+  <?php foreach ($todo as $item): ?>
+  <li><?=$item?></li>
+  <?php endforeach; ?>
+  </ul>
+
+Notice that there are no ``braces``. Instead, the end brace is replaced with
```
Comment (ravage84) [inline en/views.rst]: I don't think braces needs to be escaped.

Comment (ravage84) [inline en/views.rst]: the end brace for the ``foreach``

File: en/views.rst [hunk 3]
```diff
@@ -66,15 +66,59 @@ View Templates
...
+``endforeach``. Each of the control structures listed above has a similar
+closing syntax: ``endif``, ``endfor``, ``endforeach``, and ``endwhile``.
+
+Also notice that instead of using a ``semicolon`` after each structure
```
Comment (ravage84) [inline en/views.rst]: I would move that before taling about the end :smirk_cat:

File: en/views.rst [hunk 4]
```diff
@@ -66,15 +66,59 @@ View Templates
...
+  <?php if ($username === 'sally'): ?>
+     <h3>Hi Sally</h3>
+  <?php elseif ($username === 'joe'): ?>
+     <h3>Hi Joe</h3>
+  <?php else: ?>
+     <h3>Hi unknown user</h3>
+  <?php endif; ?>
+
+If you'd prefer using a templating language like
+`Twig <http://twig.sensiolabs.org>`_, a subclass of View will bridge your templating
```
Comment (ravage84) [inline en/views.rst]: We could as far as recommending https://github.com/WyriHaximus/TwigView, if it is any good. I cannot say, as I have no experience with.

**Comment by jeffblack360** [comment]
Thanks for reviewing everything @ravage84! 
I'll make the edits and run them past everyone again.

File: en/views.rst (hunk 4 above)
Comment (ionas) [inline en/views.rst]: I'd not link to specific plugins that are not at least part of FriendsOfCake or maybe Muffin and/or endorsed by @lorenzo or @markstory - The reason is not that those plugins are not good, the reason is the question of maintenance and updating those Plugins. Recommending them from the book pretends that that is a given.

I'd rather link to the cakephp awesome plugin list where one might (or might not) find a view template plugin.

File: en/views.rst [hunk 5]
```diff
@@ -66,16 +66,60 @@ View Templates
...
+language and CakePHP. The `TwigView plugin for CakePHP <https://github.com/WyriHaximus/TwigView>`_
+may be helpful.
```
Comment (jeffblack360) [inline en/views.rst]: Okay, I'll yank the reference to TwigView. Thank you!

**Comment by jeffblack360** [comment]
If anyone thinks there should be any additions or edits please let me know. Thanks!

File: en/views.rst (hunk 4 above)
Comment (markstory) [inline en/views.rst]: I think recommending Twig is fine without a specific plugin for now at least.

**Comment by saeideng** [comment]
i don't like these changes :)
We have a low level of education of php Syntax  in this PR changes
I suggest revert this PR

**Comment by jeffblack360** [comment]
@saeideng Well you may be correct. My hope was to assist new developers with a few examples and demonstart how templates are constructed. Thanks for the feedback!

**Comment by saeideng** [comment]
Thanks
but,i think if we(in cookbook) have education of how programing in Twig or other template Engine , that is better
however how we can setup  template Engines ...
I read the instructions but were confusing for me.
I did not expect this instruction In the middle of topics related to Cake 
you can simply just link to `http://php.net/manual/en/control-structures.alternative-syntax.php`

**Comment by jeffblack360** [comment]
@saeideng Fair enough. I do think it would be helpful to developers -- both old and new -- to illustrate the default way of constructing CakePHP templates. Maybe a good place to show off Twig would be in one the tutorials? Thoughts from the community?

**Comment by ionas** [comment]
I don't think we should encourage Twig (or any other additional templating language) as long as the core has made up their mind to actually support one. It should not be part of the book. Feel free to setup a blog, post answers on stackoverflow and help people out on IRC on the matter, or just help https://github.com/WyriHaximus/TwigView - The issue is - again - neither Twig nor the TwigView plugin but IMHO the focus of the book.

I am fine teaching our own quick template hacks that are used within bake and I am fine helping out newbies in regards to alternative control structures, though even there, aside maybe some conventions I'd simply link to the php docs for those alt structs and be DRY.

But @markstory merged it - thus I'd rather improve than discard entirely.

**Comment by jeffblack360** [comment]
@ionas Thanks for the feedback! I guess I do have a habit of being less DRY when it comes to documentation, illustrations, and examples. At least I think I do. I certainly don't believe in being a one stop shop for everything and anything in the documentation; however I do feel that I'm guilty of looking for opportunities to add clarity about the Cake "Way" of doing things.

File: en/views.rst (hunk 4 above)
Comment (lorenzo) [inline en/views.rst]: I'm on the other side, showing a community plugin will save users time researching for what they want and immediately see the power of the view layer

Comment (ionas) [inline en/views.rst]: What about adding a new icon and background color for community endorsements. In the intro we can clarify that those are non-official?

File: en/views.rst [hunk 6]
```diff
@@ -66,15 +66,58 @@ View Templates
...
+views will be rendering HTML/XHTML documents to browsers, but you might also need to
 reply to a remote application via JSON, or output a CSV file for a user.
 
-By default CakePHP template files are written in plain PHP and have a default
-extension of **.ctp** (CakePHP Template). These files contain all the
-presentational logic needed to get the data it received from the
-controller in a format that is ready for the audience you're
-serving to. If you'd prefer using a templating language like
-`Twig <http://twig.sensiolabs.org>`__, a subclass of View will bridge your templating
+CakePHP template files have a default extension of **.ctp** (CakePHP Template)
```
Comment (chinpei215) [inline en/views.rst]: @jeffblack360 Sorry to comment in a closed issue, but I noticed an undesirable change.

 In the above changes, a phrase `are written in plain PHP` was removed from the original text. But I think this is important for explaining what CakePHP Template is. Without this phrase, users cannot be sure whether CakePHP Template is just a plain PHP or yet another PHP-like template.

But this phrase seems to have removed intentionally. Why did you remove this phrase?

**Pull request merged**